html/components/tooltip.stories.js

```diff
@@ -0,0 +1,52 @@
+import { useEffect } from '@storybook/client-api';
+import { tooltip } from './tooltip';
+import '../utilities/polyfills/classListSVG';
+import { markdownDocumentationLinkBuilder } from '../../storybook-utilities/markdownDocumentationLinkBuilder';
+
+export default {
+  title: 'Components/Tooltip',
+  decorators: [
+    story => `<div class="sprk-o-Box">${story()}</div>`,
+  ],
+  parameters: {
+    info: `
+${markdownDocumentationLinkBuilder('toggle')}
+    `,
```

Comment:
We need some storybook documentation for Tooltip. Even if its just the link back to the gatsby page like the other components have.

Reply:
I changed this from "toggle" to "tooltips" so it should link to the correct page.